### DISCUSSION TITLE
Progetto esame  Open Source Gianluca Milani

### DISCUSSION THREAD

**Discussion by M1lo25** [deneb-alpha/Lab_open_source-2023_2024#27] [Projects]
ESSENZIALE:
-Spiegazione teorica delle licenze d'uso del software e della licenza GPL
-Introduzione teorica del commit e del bisect 
-Esecuzione di un bisect in GIT

FACOLTATIVA
-Progetto traduzione LibreOffice Help (inglese/italiano) tramite Weblate

**Comment by deneb-alpha** [comment]
Ciao @M1lo25,
La proposta di progetto va be, ti chiederei solo di integrare anche i tuoi commenti per quanto riguarda l'esperienza d'interazione con il progetto LibreOffice. In particolare, vorrei capire che tipo d'interazione hai avuto con la community LibreOffice, se la documentazione ufficiale è stata sufficiente, chiara, etc.

Insomma, a parte la contribuzione a LibreOffice vorrei capire com'è stata la tua esperienza con il progetto e la sua community.
